### PR TITLE
[2.9] [PR 245] Linux AppImage Documentation

### PR DESCRIPTION
**Pull request by mmattel**
Backport of PR #245

### CODE REVIEW

File: antora.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: desktop
-title: Desktop Client
-version: '2.9'
+title: Desktop App
+version: 'next'
```
Comment (phil-davis) [inline antora.yml:3]: should this be here?

Comment (mmattel) [reply antora.yml:3]: you are correct...
many thanks for catching 🤦‍♂️

Comment (mmattel) [inline antora.yml:3]:
```suggestion
version: '2.9'
```

Comment (mmattel) [reply antora.yml]: fixed

**Review by phil-davis — APPROVED** [review]

**Pull request merged**